reconcile/test/test_github_repo_invites.py

```diff
@@ -0,0 +1,116 @@
+from unittest.mock import MagicMock
+from reconcile import github_repo_invites
+from typing import Any, Iterable, Mapping
+
+from reconcile.utils.raw_github_api import RawGithubApi
+
+import pytest
+
+
+def test_parse_null_code_components():
+    raw_code_components = None
+    expected = github_repo_invites.CodeComponents(
+        urls=set(),
+        known_orgs=set(),
+    )
+    assert github_repo_invites._parse_code_components(raw_code_components) == expected
+
+
+def test_parse_valid_code_components():
+    raw_code_components: Iterable[Mapping[str, Any]] = [
+        {
+            "codeComponents": [
+                {
+                    "url": "org1/project1",
```

Comment:
why not using a url as usually found in app-interface here? e.g.

```suggestion
                    "url": "https://github.com/org1/project1",
```

Reply:
Absolutely! Good point!

Added more realistic urls as test input.